guides/source/action_view_overview.md

```diff
@@ -118,93 +157,38 @@ would produce something like:
 </div>
 ```
 
-Below is a full-length RSS example actually used on Basecamp:
```

Comment:
Do you think this is better left out? I think it's nice to see a real world example.

Reply:
I took it out as this is a less common type of template and didn't want readers to get lost in trying to read this xml example that isn't super relevant to understanding Action Views. Added a link to the Builder gem for more example as Carlos suggested. I think that should cover it. But if you feel strongly, let me know I can add the RSS example back in.